packages/@lwc/engine-server/src/__tests__/utils/custom-loader.js

```diff
@@ -0,0 +1,3 @@
+export function load() {
+    return Promise.resolve(); // stub
```

Comment:
```suggestion
    return Promise.resolve("stub");
```
Makes the difference between "didn't render" and "rendered but got replaced" more obvious.

Reply:
```suggestion
    return Promise.resolve({default: "stub"});
```
Or does it need to be an object?

Reply:
No, it can be anything.